build/v1/types.go

```diff
@@ -938,6 +938,11 @@ type BuildConfigSpec struct {
 	// When a BuildConfig is created, the 5 most recent failed builds are retained unless this value is set.
 	// If removed after the BuildConfig has been created, all failed builds are retained.
 	FailedBuildsHistoryLimit *int32 `json:"failedBuildsHistoryLimit,omitempty" protobuf:"varint,5,opt,name=failedBuildsHistoryLimit"`
+
+	// UseClusterCA denotes whether the builder container will use CAs from the cluster, or the CAs included in the builder image's own /etc/pki/ca-trust directory.
+	// Setting this to true will overwrite the contents of /etc/pki/ca-trust w/ the CA trust values from the cluster. Default: false
+	// +optional
+	UseClusterCA bool `json:"useClusterCA,omitempty" protobuf:"varint,6,opt,name=useClusterCA"`
```

Comment:
Protobuf tag issue - I don't think this should have varint?

```suggestion
	UseClusterCA bool `json:"useClusterCA,omitempty" protobuf:"bool,6,opt,name=useClusterCA"`
```

Reply:
I copied this from another bool field in this same file, i am more than happy to change it, but was just following precedent.

Reply:
updated in current push

Reply:
hrm...I don't see us using `protobuf"bool...` anywhere.

@deads2k @sttts is there a reason why we use `varint` for protobuf booleans?

Reply:
@smarterclayton defined the original proto for kube resources